backend/app.py

```
@@ -1673,6 +1673,119 @@ async def delete_team_mapping(
+            }
+        }
+
+    except Exception as e:
+        logger.error(f"Error submitting match for async processing: {e}", exc_info=True)
+        raise HTTPException(status_code=500, detail=f"Failed to queue match: {str(e)}")
+
+
+@app.get("/api/matches/task/{task_id}")
+async def get_task_status(
+    task_id: str,
+    current_user: dict[str, Any] = Depends(require_match_management_permission)
```

Ruff (0.14.1)

Unused function argument: `current_user`

(ARG001)